pyproject.toml

```diff
@@ -35,6 +35,12 @@ rich-click = "^1.6.0"
 tenacity = "^8.1.0"
 cerberus = "^1.3.4"
 flatdict = "^4.0.1"
+pytest = "^7.2.0"
+yappi = "^1.4.0"
+line-profiler = "^4.0.2"
+line-profiler-pycharm = "^1.1.0"
+flask-debugtoolbar = "^0.13.1"
```

Comment:
I'm seeing import failures trying to run this. Bumping this to `0.15.1` seems to resolve

```
    import flask_debugtoolbar  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ryan/.cache/pypoetry/virtualenvs/pipeline-dash-WTfjvmy7-py3.11/lib/python3.11/site-packages/flask_debugtoolbar/__init__.py", line 8, in <module>
    from werkzeug.urls import url_quote_plus
ImportError: cannot import name 'url_quote_plus' from 'werkzeug.urls'
```